lsst-epo/investigations-client
# build(codegen): extend Codegen search to server actions

## What this change does ##

Moves the auth server actions into the existing `lib/api/auth` module and extends the current directory search to include TS files

## Testing ##

- Start on fresh develop branch
- Log in as any user
- Confirm user can log in
- Run `yarn codegen` and try to log in again, user should not be able to log in
- Reset and pull this branch
- Log in as any user
- Confirm user can log in
- Run `yarn codegen` and try to log in again, this time user should be able to log in because correct schema is generated.
